projects/framework-blender/extensions/tool-configs/blender-scene-opener.yaml

```diff
@@ -0,0 +1,38 @@
+type: tool_config
+name: blender-scene-opener
+config_type: opener
+engine:
+  - type: plugin
+    tags:
+      - context
+    plugin: store_context_id
+
+  # Open the file component.
+  - type: group
+    tags:
+      - component
+    options:
+      component: snapshot
+    plugins:
+      - type: plugin
+        plugin: store_component
+      - type: plugin
+        tags:
+          - collector
+        plugin: component_path_collector
+        ui: asset_version_selector
+        options:
+          asset_type_name: scene
+          file_extensions:
+            - blend
```

Comment:
Any other file extension supported?

Reply:
not for now, but keeping it open for later.